airbyte-workers/src/main/java/io/airbyte/workers/process/ProcessFactory.java

```diff
@@ -43,12 +44,18 @@
    * @return the ProcessBuilder object to run the process
    * @throws WorkerException
    */
-  Process create(String jobId, int attempt, final Path jobPath, final String imageName, final String entrypoint, final String... args)
+  Process create(String jobId,
+                 int attempt,
+                 final StandardCheckConnectionInput input,
```

Comment:
we'd switch `StandardCheckConnectionInput` out for something more generic - was using this as a placeholder.